app/src/main/java/com/stevesoltys/seedvault/restore/install/DeviceInfo.kt

```diff
@@ -0,0 +1,27 @@
+package com.stevesoltys.seedvault.restore.install
+
+import android.content.Context
+import android.os.Build
+import com.android.internal.app.LocalePicker
+import com.stevesoltys.seedvault.R
+
+class DeviceInfo(context: Context) {
+    val densityDpi: Int = context.resources.displayMetrics.densityDpi
+    val supportedABIs: List<String> = Build.SUPPORTED_ABIS.toList()
+    private val deviceName: String = "${Build.MANUFACTURER} ${Build.MODEL}"
+    private val languages = LocalePicker.getSupportedLocales(context)
```

Comment:
We could add a comment here about what we based this on to make future updates easier

Reply:
What do you mean?

Reply:
Nevermind.